prime-dotnet-webapi/Migrations/20211112213455_AddedPaperSubmissionTemplate.cs

```diff
@@ -0,0 +1,85 @@
+
+            migrationBuilder.UpdateData(
+                table: "EmailTemplate",
+                keyColumn: "Id",
+                keyValue: 12,
+                column: "Template",
+                value: "<p> The site you registered in PRIME, @Model.DoingBusinessAs, has been approved by the Ministry of Health. Your SiteID is @Model.Pec. </p> <p> Health Insurance BC has been notified of the site’s approval and will contact your software vendor. Your vendor will complete any remaining setup for your site and may contact you or the PharmaNet Administrator at your site. </p> <p> If you need to update any information in PRIME regarding your site, you may log in at any time using your mobile BC Services Card. If you have any questions or concerns, please phone 1-844-397-7463 or email <a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a>. </p> <p> Thank you. </p>");
+
+            migrationBuilder.UpdateData(
+                table: "EmailTemplate",
+                keyColumn: "Id",
+                keyValue: 16,
+                column: "Template",
+                value: "Thank you for registering your site (SiteID: @Model.Pec) in PRIME. If you need to update any site information in PRIME, you may log in at any time using your mobile BC Services Card. If you have any questions, please phone 1 - 844 - 397 - 7463 or email <a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a>. Thank you.");
+
+            migrationBuilder.InsertData(
+                table: "EmailTemplate",
+                columns: new[] { "Id", "CreatedTimeStamp", "CreatedUserId", "EmailType", "ModifiedDate", "Template", "UpdatedTimeStamp", "UpdatedUserId" },
+                values: new object[] { 17, new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), new Guid("00000000-0000-0000-0000-000000000000"), 17, new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), "Your request for PharmaNet access has been approved and recorded in PRIME. When it is possible for you to do so, you must enrol in PRIME using your mobile BC Services Card. <br> <br> <strong> Your temporary GPID is @Model.GPID. </strong> <br> <br> The first time you log into PRIME you should be asked if you have previously received permission to access PharmaNet via an offline process. If you do not see this prompt, please stop your enrollment and contact <a href=\"mailto:PRIMEsupport@gov.bc.ca", new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), new Guid("00000000-0000-0000-0000-000000000000") });
```

Comment:
The email template seems to be cut off and became incomplete as `<a href=\"mailto:PRIMEsupport@gov.bc.ca"`

Not rendering correctly in mailhog
![image](https://user-images.githubusercontent.com/58282608/141598653-96949128-cc5a-45c5-9888-c67c4a7f2014.png)

Reply:
Fixed

Reply:
> Fixed
After running database migration, the email template in DB is still corrupted. see below:

![image](https://user-images.githubusercontent.com/58282608/141599747-034b0531-d80a-4a24-8dab-a318b3136f69.png)

Reply:
I did not push my changes yet

Reply:
No it's updated